cmd/mailchain/internal/http/server.go

```diff
@@ -77,6 +77,7 @@ func CreateRouter(s *settings.Root, cmd *cobra.Command) (http.Handler, error) {
 	api.HandleFunc("/nameservice/address/{address}/resolve", handlers.GetResolveAddress(config.addressResolvers)).Methods("GET")
 	api.HandleFunc("/nameservice/name/{domain-name}/resolve", handlers.GetResolveName(config.domainResolvers)).Methods("GET")
 
+	api.HandleFunc("/envelope", handlers.GetEnvelope()).Methods("GET")
```

Comment:
Can you move this up placing it in alphabetical order e.g. just below `api/addresses` it will make it easier to find in the future